java/client/src/main/java/glide/api/logging/Logger.java

```diff
@@ -28,13 +28,13 @@
 public final class Logger {
     @Getter
     public enum Level {
-        DISABLED(-2),
```

Comment:
why changing it and not using the existing code? setting disabled with -2 allows you to use `if (!(level.getLevel() <= loggerLevel.getLevel())) {` in the `log` function without checking directly if logger.Off

Reply:
In the log, it was already existing the specific check. 
In the other, I don't want to check, and I want it to be consistent over all clients. “Special” implantation for one language are not a thing we do. The disabled in java only was supposed to be blocked when it was created so it is implemented for all clients, as I do now, even if the request to fix is for ts only.
Adding a negative number will bring major changes to all other clients, and I think consistency is better than one more if.

Reply:
And about naming, the tracing crate we use, use off so I want to be consistent.